### DISCUSSION TITLE
bamboo door

### DISCUSSION THREAD

**Discussion by b1zeyofficial** [diamond-gold/DummyItemsBlocks#11] [Ideas]
This plugin doesn't seem to have a bamboo door, I have a suggestion to add it.

**Comment by diamond-gold** [comment]
It has already been added
https://github.com/diamond-gold/DummyItemsBlocks/blob/main/resources/config.yml#L9
https://github.com/diamond-gold/DummyItemsBlocks/blob/main/resources/config.yml#L119

**Comment by b1zeyofficial** [comment]
then can you fix the shield? so that it works like in the plugin
Shield author is JavierLeon9966

**Reply by diamond-gold** [reply]
Sorry I do not understand? What is wrong?
